packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,144 @@
+import { Asset } from './Asset';
+
+# Asset
+
+The Asset component is a visual representation of an asset file such as a video or a PDF document.
```

Comment:
Maybe we could come up with some examples where this component can be used? Or we consider it as a subcomponent only - if so, maybe it's worth mentioning which component uses it?